.github/workflows/pull-request-test.yml

```diff
@@ -105,43 +131,27 @@ jobs:
           amalthea: "${{ needs.check-deploy.outputs.amalthea }}"
           amalthea_sessions: "${{ needs.check-deploy.outputs.amalthea-sessions }}"
           extra_values: "${{ needs.check-deploy.outputs.extra-values }}"
-      - name: Check existing renkubot comment
-        if: needs.check-deploy.outputs.pr-contains-string == 'true'
-        uses: peter-evans/find-comment@v3
-        id: findcomment
-        with:
-          issue-number: ${{ github.event.pull_request.number }}
-          comment-author: "RenkuBot"
-      - name: Create comment pre deploy
-        if: ${{ steps.findcomment.outputs.comment-id == 0 &&
-          needs.check-deploy.outputs.pr-contains-string == 'true' }}
-        uses: peter-evans/create-or-update-comment@v4
-        with:
-          token: ${{ secrets.RENKUBOT_GITHUB_TOKEN }}
-          issue-number: ${{ github.event.pull_request.number }}
-          body: |
-            You can access the deployment of this PR at https://ci-renku-${{ github.event.number }}.dev.renku.ch
 
-  test-pr:
+  legacy-scala-tests:
     name: Legacy Scala tests
-    if: github.event.action != 'closed' && needs.check-deploy.outputs.pr-contains-string == 'true' && needs.check-deploy.outputs.test-enabled == 'true'
-    needs: [check-deploy, deploy-pr]
     runs-on: ubuntu-24.04
+    needs: [check-deploy, deploy-pr]
+    if: github.event.action != 'closed' && needs.check-deploy.outputs.pr-contains-string == 'true' && needs.check-deploy.outputs.test-legacy-enabled == 'true'
     steps:
-      - uses: SwissDataScienceCenter/renku-actions/test-renku@v1.14.1
+      - uses: SwissDataScienceCenter/renku-actions/test-renku@v1.14.2
         with:
           kubeconfig: ${{ secrets.RENKUBOT_DEV_KUBECONFIG }}
           renku-release: ci-renku-${{ github.event.number }}
           gitlab-token: ${{ secrets.DEV_GITLAB_TOKEN }}
           s3-results-access-key: ${{ secrets.ACCEPTANCE_TESTS_BUCKET_ACCESS_KEY }}
           s3-results-secret-key: ${{ secrets.ACCEPTANCE_TESTS_BUCKET_SECRET_KEY }}
-          test-timeout-mins: "120"
+          test-timeout-mins: "60"
 
   legacy-cypress-acceptance-tests:
     name: Legacy Cypress tests
-    needs: [check-deploy, deploy-pr]
     runs-on: ubuntu-24.04
-
+    needs: [check-deploy, deploy-pr]
+    if: github.event.action != 'closed' && needs.check-deploy.outputs.pr-contains-string == 'true' && needs.check-deploy.outputs.test-legacy-enabled == 'true'
```

Comment:
issue: I am pretty sure this isn't needed because it shows up in line 160 below.

Reply:
Oh actually, the other one is wrong since it doesn't use `test-legacy-enabled`.

Fixed in https://github.com/SwissDataScienceCenter/renku/pull/3948/commits/1564b71abadd24b3ce2f6f9838c1f3d9548eb75c